_includes/templates/compatibility-page.md

```diff
@@ -0,0 +1,210 @@
+  - **Sent transaction labeled replaceable in transaction details**<br>
+    Visually indicates that a sent transaction has signaled RBF when viewing the transaction details
+  {% when "false" %}{:.feature-bad}
+  - **Sent transaction not labeled replaceable in transaction details**<br>
+    Does not visually indicate that a sent transaction has signaled RBF when viewing the transaction details
+  {% when "na" %}{:.feature-neutral}
+  - **Does not show transaction details**<br>
+    This service does not show transaction details natively. Usually this means the service links off to a block explorer for transaction details.
+  {% when "untested" %}{:.feature-neutral}
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
+  {% else %}{% include ERROR_42_UNEXPECTED_VALUE %}
+{% endcase %}
+
+{:id="send-replaced"}
+{% if rbf.send.shows_replaced_version == "true" and rbf.send.shows_original_version == "true"  %}
+  {:.feature-yes}
+  - **Shows replacement and original transactions**<br>
+    Both the original transaction and replacement transactions are shown in the
+    transaction list
+{% elsif rbf.send.shows_replaced_version == "na" or
+    rbf.send.shows_original_version == "na" %}
+  {:.feature-neutral}
+  - **No replacements in transaction list**<br>
+    Because no transaction replacement is possible, we could not test whether
+    original or replacement sent transactions are shown after replacement
+{% elsif rbf.send.shows_replaced_version == "untested" or
+    rbf.send.shows_original_version == "untested" %}
+  {:.feature-neutral}
+  - **Not tested**<br>
+    We either didn’t test this or could not appropriately determine the results
+{% elsif rbf.send.shows_replaced_version == "true" %}
+  {:.feature-yes}
+  - **Shows replacement transaction only**<br>
+    Only the replacement transaction is shown in the transaction list. No original
+    transaction shown
+{% elsif rbf.send.shows_original_version == "true" %}
+  {:.feature-bad}
+  - **Shows original transaction only**<br>
+    Only the original transaction shown in transaction list. Repacement transactions
+    not shown
+{% elsif rbf.send.shows_original_version == "false" and rbf.send.shows_replaced_version == "false" %}
+  {:.feature-bad}
+  - **No unconfirmed transactions**<br>
+    Neither the original nor replacement transactions are shown in the
+    transaction list. Most likely unconfirmed transactions are not supported
+{% else %} {% include ERROR_42_UNEXPECTED_VALUE %}
+{% endif %}
+
+### Usability
+
+*Click on a thumbnail for a larger image or to play its video.*
+
+{% for example in tool.rbf.examples %}{% capture /dev/null %}
+  {% if example.link %}
+    {% assign link = example.link %}
+  {% else %}
+    {% assign link = example.image %}
+  {% endif %}
+{% endcapture %}
+<div markdown="1" class="compat-usability">
+[![{{example.caption|escape_once}}]({{example.image}}){:width="250px"}]({{link}})
```

Comment:
Nit: this `{:width="250px"}` should probably be changed to a class and the width defined in the CSS.